src/pages/signalProviders/index.js

```diff
@@ -43,6 +48,15 @@ const SignalProviders = (props) => {
   const idIndex = process.env.GATSBY_BASE_PATH === "" ? 2 : 3;
   const providerId = location.pathname.split("/")[idIndex];
   const dispatch = useDispatch();
+  const emptySettings = creatEmptySettingsEntity();
+  const { setEmptySettingsAlert } = useContext(AppContext);
+  const [settings, setSettings] = useState(emptySettings);
+  const quoteAssets = useSelectedExchangeQuotes(selectedExchange.internalId);
+  const quotes =
+    selectedExchange.name.toLowerCase() === "bitmex"
+      ? { BTC: { quote: "BTC", minNotional: 0 } }
+      : quoteAssets;
+  const quotesAvailable = Object.keys(quotes).length > 0 ? "true" : "false";
```

Comment:
If it's a boolean let's use `true` and `false`

Reply:
Done